response/settings/prod.py

```diff
@@ -55,3 +55,11 @@
     INCIDENT_CHANNEL_ID = get_channel_id(INCIDENT_CHANNEL_NAME, SLACK_TOKEN)
 except:
     INCIDENT_CHANNEL_ID = None
+
+
+PAGERDUTY_ENABLED = os.getenv("PAGERDUTY_ENABLED") in ("True", "\"True\"", "true", "\"true\"", True, 1)
+if PAGERDUTY_ENABLED:
+    print("PagerDuty is Enabled")
+    PAGERDUTY_API_KEY = get_env_var("PAGERDUTY_API_KEY")
+    PAGERDUTY_SERVICE = get_env_var("PAGERDUTY_SERVICE")
+    PAGERDUTY_DEFAULT_EMAIL = get_env_var("PAGERDUTY_DEFAULT_EMAIL")
```

Comment:
Can you add a newline to make github happy?

Reply:
@adamlphillips Added.